test/verify/check-metrics

```diff
@@ -626,6 +626,15 @@ class TestMultiCPU(MachineCase):
         self.assertGreaterEqual(getMaximumSpike(b, "cpu", False, 1598968800000, 45), 0.5)
         self.assertLessEqual(getMaximumSpike(b, "cpu", False, 1598968800000, 45), 1.0)
 
+        # Test current usage of cores
+        b.wait(lambda: progressValue(b, "#current-cpu-usage") < 20)
+        m.execute("systemd-run --collect --slice cockpittest -p CPUQuota=60% --unit cpu-hog dd if=/dev/urandom of=/dev/null")
+        m.execute("systemd-run --collect --slice cockpittest -p CPUQuota=30% --unit cpu-piglet dd if=/dev/urandom of=/dev/null")
+        b.wait(lambda: progressValue(b, "#current-cpu-usage") > 40)
+        b.mouse("#current-cpu-usage", "mouseenter")
+        b.wait(lambda: int(b.text(".pf-c-tooltip .pf-l-flex:first-child div:nth-child(2)")[:-1]) > 50)
+        b.wait(lambda: int(b.text(".pf-c-tooltip .pf-l-flex:first-child div:nth-child(2)")[:-1]) > 20)
```

Comment:
Is that some copy&paste error? don't you mean `nth-child(1)` in the first line? These selectors look exactly identical right now.